__tests__/api.test.js

```diff
@@ -167,3 +167,86 @@ describe('GET/api/articles/:article_id/comments', () => {
         })
     });
 });
+
+
+describe('POST /api/articles/:article_id/comments', () => {
+    it('201: responds with a status of 201', () => {
+        const newComment = {
+            username: "butter_bridge",
+            body: "Hello"
+        };
+
+        return request(app).post('/api/articles/4/comments').send(newComment)
+        .expect(201);
+    });
+    
+    it('201: responds with a comment object that has been sent', () => {
+        const newComment = {
+            username: "butter_bridge",
+            body: "Hello"
+        };
+
+        return request(app).post('/api/articles/4/comments').send(newComment)
+        .expect(201)
+        .then((response) => {
+            const {comment} = response.body;
+            expect(comment.author).toEqual(newComment.username);
+            expect(comment.body).toEqual(newComment.body);
+        })
+    });
+
+    it('400: responds with an error message if username or body is missing', () => {
+        const invalidComment = {
+            username: "butter_bridge"
+        };
+        
+        return request(app).post('/api/articles/4/comments').send(invalidComment)
+        .expect(400)
+        .then((response) => {
+            const {message} = response.body;
+            expect(message).toBe('400: Bad Request')
+        })
+    });
+
+    it('400: responds with an error message if username or body is empty', () => {
+        const invalidComment = {
+            username: "",
+            body: "Hello"
+        };
```

Comment:
This test is good but the description says if the body or username is empty, you are only testing one of them so your description should be more precise. Also this could be a test for if username isn't in the database.